app/views/tags/wrangle.html.erb

```diff
@@ -140,8 +140,13 @@
                 <% unless params[:status] == 'unwrangled' %>
                   <li>
                     <label for="remove_associated_<%= tag.id %>">
-                      <%= ts('Remove') %>
-                      <%= check_box_tag 'remove_associated[]', tag.id, false, id: "remove_associated_#{tag.id}" %>
+                      <% if tag.unwrangleable? && tag.parents.by_type("Fandom").size == 1 %>
+                        <%= ts('Unwrangleable') %>
```

Comment:
Also, we have plans to add a checkbox for marking tags unwrangleable from the bins, so it could cause confusion there as well. 

It's probably best to take the "Remove" option away when it can't be used. That's what we do on fandoms' Unwrangled bins.

Reply:
My bad, I changed the label for my own debugging and didn't set it back.  We could just hide the button entirely, but does that also raise the same question about users wondering why it's missing?

Reply:
I don't think so -- as I said, it's what we do on the unwrinkled page. However, adding the class `disabled` to the `label` and disabling the checkbox will also work fine.

Reply:
...wow, really, autocorrect? The _unwrangled_ page. I'm not sure what the unwrinkled page is.